### PR TITLE
Remove groups

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala [hunk 1]
```diff
@@ -154,15 +154,10 @@ class ProcessValidation(validators: ProcessingTypeDataProvider[ProcessValidator]
...
 
   private def validateDuplicates(displayable: DisplayableProcess): ValidationResult = {
-    val groupIds = displayable.metaData.additionalFields
-      .flatMap(a => a.cast[ProcessAdditionalFields])
-      .toList
-      .flatMap(_.groups)
-      .map(_.id)
     val nodeIds = displayable.nodes.map(_.id)
 
     //in theory it would be possible to have group named like one of nodes inside, but it's not worth complicating logic...
```
Comment (dswiecki) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala:159]: hmm it this comment still valid ? even more, can we have duplicates right now ?

File: ui/client/reducers/graph/reducer.ts [hunk 2]
```diff
@@ -322,7 +283,7 @@ const reducer = mergeReducers(
     processToDisplay: {
       nodes,
       properties: {
-        additionalFields: {groups},
+        additionalFields: {},
```
Comment (dswiecki) [inline ui/client/reducers/graph/reducer.ts:286]: is explicit setting additionalFields still required ?

**Review by dswiecki — APPROVED** [review]

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala (hunk 1 above)
Comment (mproch) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala:159]: :+1: comment is not valid, but we can have duplicates defined in FE

File: ui/client/reducers/graph/reducer.ts (hunk 2 above)
Comment (mproch) [reply ui/client/reducers/graph/reducer.ts:286]: I'd leave it for next refactor....

**Pull request merged**